tests/testing/LocalBearTestHelperTest.py

```diff
@@ -0,0 +1,43 @@
+from queue import Queue
+import unittest
+
+from tests.test_bears.TestBear import (FirstTestBear,
+                                       SecondTestBear,
+                                       ThirdTestBear)
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import verify_local_bear, execute_bear
+
+
+files = (("Everything is invalid/valid/raises error",),)
```

Comment:
https://github.com/coala-analyzer/coala/pull/2293/files#diff-ca216335ff62d78f276a93743cd88ff7R24

Initialize `files` as `self.files` and use that in lines 34, 38, 42?